backend/app/config.py

```
@@ -43,6 +44,29 @@ def get_fred_api_key() -> str:
+        try:
+            entry = (
+                db.query(AppSetting)
+                .filter(AppSetting.key == "slack_webhook_url")
+                .first()
+            )
+            if entry:
+                return entry.value
+        finally:
+            db.close()
+    except Exception:
+        pass
```

Ruff (0.15.6)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)